### PR TITLE
feat: Jest Configurator tests for error view

### PR DESCRIPTION
**Pull request by jimhunty**
Adding jest configurator tests to the Error View package.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c12b2dbf40fc7cc048d34b0d2cf740edf757836f/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/eaf1bf5fa01cfc01dc3f609ea02df313a4b4da1e/dextrose-test/index.html

File: packages/error-view/__tests__/android/error-view.android.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import shared from "../shared";
+
+describe("Article test on android", () => {
```
Comment (colincclark) [inline packages/error-view/__tests__/android/error-view.android.test.js]: Should be `Error View test.....`?

Comment (jimhunty) [reply packages/error-view/__tests__/android/error-view.android.test.js]: Whoops! Now sorted. Ready for re-review.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e7252150c17c288127e5696cc648851ceb126d89/dextrose-test/index.html

**Pull request merged**